Cargo.toml

```diff
@@ -21,9 +21,9 @@ codegen-units = 1
 mimalloc = { version = "0.1.29", default-features = false }
 clap = { version = "3.2.8", features = [ "derive", "wrap_help" ] }
 regex = { version = "1.5.6", default-features = false, features = [ "std", "unicode-perl" ] }
-rusqlite = { version = "0.26.3", default-features = false }
 colored = "2.0.0"
 ureq = { version = "2.4.0", default-features = false, features = [ "native-tls", "json" ] }
 serde = { version = "1.0.138", default-features = false, features = [ "derive", "serde_derive" ] }
 native-tls = { version = "0.2.10", default-features = false }
 libc = { version = "0.2.126", default-features = false }
+rm_rf = { version = "0.6.2", default-features = false }
```

Comment:
Does it really need a whole crate just for `rm -rf`? couldn't this be made a function in amethyst directly?

Reply:
This is actually because rm_rf handles removing a directory with all subdirs in a way that doesn't crawl the entire tree (Which is what caused the permissions issue with fakeroot and failed build caches earlier) in a platform-agnostic way (Since I'm planning to maintain MSYS2 compat)

Reply:
Plus, it's less code for us to maintain. `cargo audit` shows no security vulnerabilities, so it's safe to use it for that purpose

Reply:
If it has no issues and gives less code maintaining work it's fine by me